docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
So how will we know if it didn't work? Should I add some check to the script to make sure if worked? I can just print the pint version number and the script will fail if pint was not installed?